### PR TITLE
Update website to show CFP has closed

### LINKED ISSUE #109: Homepage: Replace ellipses with text-mix
See `index.md`, line 11.

### PR DESCRIPTION
**Pull request by dcwalk**
Quick tuck to website to:

- [x] update language around CFP
- [x] show only upcoming open hours
- [x] fix #109 and add in text-mix 
- [x] bios (oops, my cherry-picking got the best of me)
  - [x] @dcwalk 
  - [x] @benhylau

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ _We are grateful to the [First Nations House](https://www.studentlife.utoronto.c
 
 Our Networks organizers have hosted civic tech events and community networks workshops, created network literacy materials, and built mesh networking software for the past two years.
 
-**Benedict Lau**{:#benhylau} is an engineer working on mobile software and mesh networks. He is a contributor and organizer at [Toronto Mesh](https://tomesh.net/), currently focused on meshing with single-board computers and building deployment tools and literacy around peer-to-peer applications.
+**Benedict Lau**{:#benhylau} is an engineer telling stories of technology practices that bring communities together. He studies distributed protocols and collective governance of digital infrastructures. When he isn't "on emails", he builds passable open source tools and facilitates activities about building peer-to-peer local networks as a way to co-imagine a future equitable web. He is a member of the Hypha Worker Co-operative and core contributor at [Toronto Mesh](https://tomesh.net).
```
Comment (dcwalk) [inline about.md]:
```suggestion
**Benedict Lau**{:#benhylau} is an engineer who tells stories of technology practices that bring communities together. He studies distributed protocols and collective governance of digital infrastructures. When not "on email", he builds passable open source tools and facilitates activities about peer-to-peer local networks as a way to co-imagine a future equitable web. He is a member of the Hypha Worker Co-operative and a core contributor at [Toronto Mesh](https://tomesh.net).
```

(copying over from https://github.com/ournetworks/ournetworks.ca/pull/106/commits/271f8409d672c0866dc4039ea5d895cc7efb24a3#r299007713)

**Comment by dcwalk** [comment]
Erroring on dat clone ^^, any insight @garrying?

**Comment by garrying** [comment]
The archive got into a stuck state. Fixed now

**Comment by dcwalk** [comment]
Thanks! Thoughts on other bits?

**Review by garrying — APPROVED** [review]
Spotted an issue with text-mix causing unexpected line-breaks while transitioning between words. Fixed in ece7768

**Comment by dcwalk** [comment]
Okay, just waiting on @benhylau RE: bio edits, thanks!

**Review by mylonelycomputer — APPROVED** [review]
Looks good to me 👍🏼

**Pull request merged**